usr/sbin/p11sak/p11sak.c

```diff
@@ -13,12 +13,61 @@
 #include <stdio.h>
 #include <string.h>
 #include <pkcs11.h>
+#include <dlfcn.h>
```

Comment:
Squash this commit also with your very first commit. Again: Why fixing your own commit, if you can do it right in the first commit?

Reply:
done